pkg/components/components.go

```diff
@@ -63,8 +63,8 @@ func (ai *AddonsImages) FillDefaults() *AddonsImages {
 	return ai
 }
 
-func GetDeployment(version string, operatorVersion string, namespace string, repository string, tag string, imagePullPolicy string, addonsImages *AddonsImages) *appsv1.Deployment {
-	image := fmt.Sprintf("%s/%s:%s", repository, Name, tag)
+func GetDeployment(version string, operatorVersion string, namespace string, repository string, name string, tag string, imagePullPolicy string, addonsImages *AddonsImages) *appsv1.Deployment {
+	image := fmt.Sprintf("%s/%s:%s", repository, name, tag)
```

Comment:
Here you use `name` for the image, in the rest of the PR you use it for container. What is the relation between the two?